index.html

```diff
@@ -302,7 +302,7 @@ <h2>Securing JSON-LD VCs with COSE</h2>
         </p>
         <p>[[rfc8152]] MAY be used to secure this media type.</p>
         <p>When using this approach, the <code>type (TBD)</code> MUST be
-          <code>vc+ld+cwt</code>
+          <code>vc+ld+json+cose</code>
```

Comment:
```suggestion
          <code>vc+ld+cose</code>
```
possible suggestion.... not worth blocking PR over, and not clear which would be better